RELEASE-NOTES.txt

```diff
@@ -1,3 +1,7 @@
+> [***] High Priority, e.g., Major Enhancements, Widespread Crash Fixes, etc.
+> [**] Medium Priority, e.g., Minor Enhancements Users Will Notice, etc.
+> [*] Low Priority, e.g., Minor Enhancements Users May Miss, etc.
```

Comment:
Reading "user may miss" made me think that we'd then want to let them know about it in the release notes, but that clashes with what "low priority" tells me 🤔 

I wonder if that's just my perception being distorted by all the easter eggs videos I've been watching on YouTube, or if there's something we can do to align the language.

<img width="1459" alt="image" src="https://user-images.githubusercontent.com/1218433/227854549-47fd4968-a166-45c7-9e4a-d942a428c8fe.png">